test/dotnet-new3.UnitTests/Approvals/AllWebProjectsWork.CanShowHelp_Mvc.approved.txt

```diff
@@ -1,8 +1,22 @@
 ﻿ASP.NET Core Web App (Model-View-Controller) (C#)
 Author: Microsoft
-Description: A project template for creating an ASP.NET Core application with example ASP.NET Core MVC Views and Controllers. This template can also be used for RESTful HTTP services.
+Description: A project template for creating an ASP.NET Core application with example ASP.NET Core MVC Views and Controllers. This template can also be used 
```

Comment:
there's an odd line break here, is this just because of the line width difference between the old and the new formats?  When looking at these tests, is there a given line width to keep the output consistent?

Reply:
I'll check it. `HelpBuilder` has `MaxWidth` however it should be used when the output is redirected. These also may change with alternative help customization approach.

Reply:
not applicable anymore